import-service/src/__tests__/importFileParser.test.ts

```diff
@@ -0,0 +1,130 @@
+import { S3Event } from 'aws-lambda';
+import { S3Client, GetObjectCommand, CopyObjectCommand, DeleteObjectCommand } from '@aws-sdk/client-s3';
+import { mockClient } from 'aws-sdk-client-mock';
+import { Readable } from 'stream';
+import { handler } from '../importFileParser';
+
+const s3ClientMock = mockClient(S3Client);
+
+describe('importFileParser lambda', () => {
+  beforeEach(() => {
+    s3ClientMock.reset();
+  });
+
+  it('should process CSV file from S3', async () => {
+    // Mock S3 event
+    const mockEvent: S3Event = {
+      Records: [{
+        s3: {
+          bucket: {
+            name: 'test-bucket'
+          },
+          object: {
+            key: 'uploaded/test.csv'
+          }
+        }
+      }]
+    } as any;
+
+    // Create a mock readable stream with CSV content
+    const mockCsvContent = 'id,title,price\n1,Test Product,10.99';
+    const mockStream = Readable.from([mockCsvContent]);
+
+    // Mock S3 GetObjectCommand response
+    s3ClientMock
+      .on(GetObjectCommand).resolves({
+        Body: mockStream
```

Comment:
This fails with an error `error TS2322: Type 'Readable' is not assignable to type 'StreamingBlobPayloadOutputTypes | undefined'.`
```suggestion
        Body: mockStream as any
```